app/forms/pages/conditions_form.rb

```diff
@@ -64,4 +65,11 @@ def assign_skip_to_end
       self.skip_to_end = false
     end
   end
+
+private
+
+  def pages_after_current_page(all_pages, current_page)
+    current_index = all_pages.index(current_page)
+    current_index ? all_pages.slice(current_index + 1, all_pages.length - current_index - 1) : []
```

Comment:
Think you could potentially make use of `filter` here:
```suggestion
    all_pages.filter{|page| page.position > current_page.position}
```